### ISSUE TITLE
admin order and custom options on simple product

### ISSUE THREAD

**Issue by fredmrn** [organicinternet/magento-configurable-simple#120]
When you try to create an order in the admin (order by phone), it seems there is no possibility to have access to the customs options of the simple product. No javascript load like in the front office. 
Is there someone can help me to do it ?

**Comment by 3poo** [comment]
I have cecked at our server, all works fine if you add a product via "Add Products" and select the simple product. If you put products from the left side (wishlist, etc) into order, then dont show the custom options!

**Comment by fredmrn** [comment]
you're right, The problem is when you select a configurable product in the left side. 
Perhaps, for a better experience, it would be best to select the configurable product and allow to access dynamically to customs options of the simple product (like in the front).

**Comment by Priyajeet34** [comment]
I'm facing the same problem
